### ISSUE TITLE
Barrier Block Problem

### ISSUE THREAD

**Issue by Dsa-123** [SkriptLang/Skript#6774]
### Skript/Server Version

```markdown
[23:18:58 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[23:18:58 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[23:18:58 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[23:18:58 INFO]: [Skript] Server Version: git-Paper-550 (MC: 1.19.4)
[23:18:58 INFO]: [Skript] Skript Version: 2.8.6 (skriptlang-github)
[23:18:58 INFO]: [Skript] Installed Skript Addons:
[23:18:58 INFO]: [Skript]  - skript-advancements v1.6.2
[23:18:58 INFO]: [Skript] Installed dependencies:
[23:18:58 INFO]: [Skript]  - Vault v1.7.3-b131
[23:18:58 INFO]: [Skript]  - WorldGuard v7.0.7+216b061
```

### Bug Description

Adding barrier block to a local variables list result in adding a particle

### Expected Behavior

Expected to add a barrier block instead of `block marker` particle

### Steps to Reproduce

```on load:
	add stone to {_items::*}
	add barrier to {_items::*}
	send {_items::*} to console```

### Errors or Screenshots

[23:18:42 INFO]: stone block
[23:18:42 INFO]: block marker

### Other

_No response_

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by sovdeeth** [comment]
Should be fixed by #6760 
Try the nightly here: https://github.com/SkriptLang/Skript/actions/runs/9357528734

**Comment by Dsa-123** [comment]
It seems to work with that nightly. Thank you!

**Comment by APickledWalrus** [comment]
This was resolved in 2.8.7
